source/scripts/notesEditor.js

```diff
@@ -69,13 +63,21 @@ function addEditToggle(preview, id){
         editButton.innerHTML = 'Edit';
     } else {
```

Comment:
This if-else statement is unnecessary. Lines 64-67 will never run because we only call addEditToggle() once in the init function and preview is always true when we call it.